types/index.d.ts

```diff
@@ -225,6 +225,7 @@ export interface Options {
 
 export interface Localization {
   body?: {
+    locale?: object; // The date-fns locale object applied to the datepickers
```

Comment:
Can you rename this as dateTimePickerLocalization, please?

Reply:
Done